src/System.Collections.NonGeneric/src/System/Collections/Hashtable.cs

```diff
@@ -755,11 +748,7 @@ public virtual ICollection Keys
         // 
         public virtual ICollection Values
         {
-            get
-            {
-                if (_values == null) _values = new ValueCollection(this);
```

Comment:
Please revert this change but do feel free to put this if block on its own line to make it match the coding guidelines.